src/main/java/rx/observables/SyncOnSubscribe.java

```diff
@@ -363,7 +373,12 @@ private boolean tryUnsubscribe() {
         }
 
         private void doUnsubscribe() {
-            parent.onUnsubscribe(state);
+            try {
+                parent.onUnsubscribe(state);
+            } catch (Throwable e) {
```

Comment:
Not really, this exception has nowhere to go and hijacking the plugins is a nightmare.